specs/experimental/validator-system-v2/validator-manager.md

```diff
@@ -0,0 +1,173 @@
+
+# Validator Manager
+
+<!-- All glossary references in this file. -->
+
+[g-validator]: ../../glossary.md#validator
+[g-l2-output]: ../../glossary.md#l2-output-root
+[output-oracle]: ../../glossary.md#l2-output-oracle-contract
+[colosseum-contract]: ../../glossary.md#colosseum-contract
+[g-validator-reward]: ../../glossary.md#validator-reward
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Validator Registration](#validator-registration)
+- [Validator Management](#validator-management)
+  - [Validator Tree](#validator-tree)
+  - [Status of Validator](#status-of-validator)
+  - [Jail](#jail)
+- [Entry Point](#entry-point)
+  - [Slashing](#slashing)
+  - [Reward Distribution](#reward-distribution)
+- [Summary of Definitions](#summary-of-definitions)
+  - [Constants](#constants)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+The Validator Manager manages the set of [validators][g-validator] and selects the next priority validator with the
+priority to submit the [output root][g-l2-output]. It is also the entry point for other contracts, such as the
+[L2 Output Oracle][output-oracle] and the [Colosseum][colosseum-contract], which distribute
+[output rewards][g-validator-reward] and slash the challenge losers. It makes successive calls to the
+[Asset Manager contract](./asset-manager.md) to apply changes to the validators' assets.
+
+## Validator Registration
+
+To become a validator, the first step is to register to the Validator Manager with at least `MIN_REGISTER_AMOUNT` of
+tokens (KRO). Upon registration, they initiate their [`Vault`](./asset-manager.md#composition-of-asset-manager) and set
+commission rate and maximum change rate of the commission rate. The commission rate is the percentage that the validator
+takes from the output reward, and the maximum change rate is the maximum range of the commission rate that the validator
+can change at one time. If `COMMISSION_RATE_MIN_CHANGE_SECONDS` has not elapsed since the last time the validator
+changed the commission rate, the validator cannot change it again. These information and safeguards can help delegators
+choose which validators to delegate to.
+
+The `Validator` struct that represents the information of the validator is defined as follows:
+
+```solidity
+struct Validator {
+    bool isInitiated;
+    uint8 noSubmissionCount;
+    uint8 commissionRate;
+    uint8 commissionMaxChangeRate;
+    uint128 commissionRateChangedAt;
+}
+```
+
+After registration, validators can be delegated tokens and KGH NFTs. When the sum of delegated tokens exceeds
```

Comment:
"delegated KRO tokens, excluding KGH NFTs"
How about something like this, and remove the "not subject to slashing" part below because I don't think it's intuitive to the reader.